package.json

```diff
@@ -12,18 +12,22 @@
     "build:conferences": "node scripts/build-conferences.js && cp conferences.json target/",
     "build:js": "rollup -c rollup.config.js",
     "test": "node scripts/test-communities.js && node scripts/test-conferences.js",
```

Comment:
Doesn't look like the unit tests are run when you issue `yarn test`.

Maybe rename `test` to `test:data` and use `npm-run-all` to run both for `test`?